js/customize-snapshots.js

```diff
@@ -765,9 +765,11 @@
 
 			// Change update button to schedule.
 			if ( component.isFutureDate() ) {
-				save.html( component.data.i18n.scheduleButton );
+				save.text( component.data.i18n.scheduleButton );
+			} else if ( api.state( 'snapshot-exists' ).get() ) {
+				save.text( component.data.i18n.updateButton );
 			} else {
-				save.html( component.data.i18n.updateButton );
+				save.text( component.data.i18n.saveButton );
```

Comment:
Yes, creating more states would be good and then listening for changes to those states in a centralized location I think is the best architecture.